app/models/moneyservicebusiness/MoneySources.scala

```diff
@@ -37,11 +37,18 @@ object MoneySources {
   import jto.validation.forms.Rules._
   import utils.MappingUtils.Implicits._
 
-  private def nameType(fieldName: String) = {
+  private def bankNameType(fieldName: String) = {
```

Comment:
because this def is not being reused anymore you can get rid of the parameter and change it to val

Reply:
Good spot changed :)